velox/docs/develop/types.rst

```diff
@@ -0,0 +1,55 @@
+=====
+Types
+=====
+
+Velox supports scalar types and complex types. These types cover both
```

Comment:
> These types cover both Presto and Spark data types.

This sentence doesn't seem accurate. Velox types cover only a subset of Presto and Spark types. There are also application-specific types like TIMESTAMP WITH TIME ZONE.

Reply:
I will update to `These types cover most Presto and Spark data types`.